ols/app/endpoints/ols.py

```diff
@@ -113,10 +120,9 @@ def conversation_request(llm_request: LLMRequest) -> LLMResponse:
                 )
 
     if config.conversation_cache is not None:
+        conversation = Conversation(llm_request.query, str(response or ""))  # type: ignore
```

Comment:
if the response is None then it doesn't seem like we should be inserting/updating anything in the cache.  but also if the response is None we should probably be returning some sort of error or placeholder response to the user.